src/lib/k6/utils.ts

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-ignore
+import { randomString as _randomString } from 'https://jslib.k6.io/k6-utils/1.2.0/index.js';
+
+export const randomString = (length = 10, charset?: string): string => {
```

Comment:
Can't this constant be removed now?

Reply:
nope, it wraps the k6 script, we don't use deno and remote imports are problematic.
unfortunately there is no `real` package.